### PR TITLE
Add `always` config for EncapsedStringsToSprintfRector

### LINKED ISSUE #8690: Change EncapsedStringsToSprintfRector to always use sprintf
I looked at the EncapsedStringsToSprintfRector rule and it seems that it chooses when to use concat or sprintf.

In our team, we'd like to always use sprintf.

Would it be OK if I send a PR to modify the EncapsedStringsToSprintfRector so that this mode can be configured? The default behavior can be the same.

### PR DESCRIPTION
**Pull request by ruudk**
Fixes https://github.com/rectorphp/rector/issues/8690

This makes it possible to always convert to sprintf.

@TomasVotruba @samsonasik Please let me know what you think. I need some inspiration for the config key.

### CODE REVIEW

File: rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/AlwaysSprintfTest.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodingStyle\Rector\Encapsed\EncapsedStringsToSprintfRector;
```
Comment (samsonasik) [inline rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/AlwaysSprintfTest.php]:
```suggestion
namespace Rector\Tests\CodingStyle\Rector\Encapsed\EncapsedStringsToSprintfRector;
```

Comment (ruudk) [reply rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/AlwaysSprintfTest.php]: Done

File: rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/FixtureAlwaysSprintf/encapsed_strings_to_sprintf_should_escape_percent.php.inc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\CodingStyle\Rector\Encapsed\EncapsedStringsToSprintfRector\Fixture;
```
Comment (samsonasik) [inline rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/FixtureAlwaysSprintf/encapsed_strings_to_sprintf_should_escape_percent.php.inc]: should use `FixtureAlwaysSprintf` subnamespace

```suggestion
namespace Rector\Tests\CodingStyle\Rector\Encapsed\EncapsedStringsToSprintfRector\FixtureAlwaysSprintf;
```

on other files in this directory as well

Comment (ruudk) [reply rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/FixtureAlwaysSprintf/encapsed_strings_to_sprintf_should_escape_percent.php.inc]: Oops, you're right. Fixed!

**Review by samsonasik — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**